contracts/monitoring/src/msg.rs

```diff
@@ -0,0 +1,16 @@
+use connection_router_api::ChainName;
+use cosmwasm_schema::{cw_serde, QueryResponses};
+use multisig::worker_set::WorkerSet;
+
+#[cw_serde]
+pub struct InstantiateMsg {}
+
+#[cw_serde]
+pub enum ExecuteMsg {}
+
+#[cw_serde]
+#[derive(QueryResponses)]
+pub enum QueryMsg {
+    #[returns(Vec<(ChainName, WorkerSet)>)]
+    GetChainActiveWorkerSet { chains: Vec<ChainName> },
```

Comment:
```suggestion
    GetActiveVerifiersForChain{ chain: ChainName },
```
we might want to add batching later, but for the beginning I'm with Sammy in keeping it simple